rewards/df-challenges.md

```diff
@@ -0,0 +1,40 @@
+---
+description: >-
+  If you are a data scientist or simply enthusiastic about extracting value from datasets 📊, we have an exciting opportunity waiting for you! Not only will you gain valuable insights and knowledge, but you'll also have the chance to compete for enticing weekly rewards. However, keep in mind that these rewards must be earned by competing with other participants! 🏆
+---
+
+# Participage in Data Farming Challenges (Red Belt)
```

Comment:
Not matching w/ SUMMARY.md
```
[DF Emissions & APYs (Red Belt)](rewards/df-emissions-apys.md)
```

df-challenges.md (should be challenge-df.md)
```
# Participage in Data Farming Challenges (Red Belt)
```

will adjust